fcomm/tests/makefile_tests.rs

```diff
@@ -1,34 +1,34 @@
-use std::env;
-use std::path::PathBuf;
-use std::process::Command;
-
-#[ignore]
-#[test]
-fn test_make_fcomm_examples() {
-    // Get the current working directory
-    let mut current_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-
-    // Find the examples directory
-    current_dir.push("examples");
-    let examples_dir = current_dir.as_path();
-    assert!(
-        examples_dir.exists() && examples_dir.is_dir(),
-        "Failed to find the fcomm examples directory"
-    );
-
-    // Run the make command in the examples directory
-    let cpus = num_cpus::get();
-
-    let make_output = Command::new("make")
-        .current_dir(examples_dir)
-        .arg(format!("-j{}", cpus))
-        .output()
-        .expect("Failed to run the make command, is make installed?");
-
-    // Check the exit status
-    assert!(
-        make_output.status.success(),
-        "Make command exited with an error: {}",
-        String::from_utf8_lossy(&make_output.stderr)
-    );
-}
+//use std::env;
+//use std::path::PathBuf;
+//use std::process::Command;
+//
+//#[ignore]
+//#[test]
+//fn test_make_fcomm_examples() {
```

Comment:
This test probably shouldn't be commented out, though it still fails on master until we get a new Nova release: https://app.circleci.com/pipelines/github/lurk-lab/lurk-rs/3137/workflows/7fd41cc0-a358-43ee-87d9-4b7196e3eb63/jobs/18845

Reply:
So the Nova release is necessary, but not sufficient to solve this. Post #499 the fcomm tests fail with a characteristic error :
``` 
huitseeker@lightning.local➜lurk-rs/fcomm/examples(upgrade-nova)» FCOMM_DATA_PATH=/tmp/nufcomm make num-list-commitment                     [20:26:08]
cargo run --release -- commit --function num-list-function.json --commitment num-list-commitment.json
    Finished release [optimized] target(s) in 0.11s
     Running `/Users/huitseeker/tmp/lurk-rs/target/release/fcomm commit --function num-list-function.json --commitment num-list-commitment.json`
thread 'main' panicked at 'committed expression read_from_path: CacheError("Cache deserialization error: invalid value: integer `2019893883`, expected variant index 0 <= i < 2")', fcomm/src/bin/fcomm.rs:191:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
make: *** [num-list-commitment] Error 101
```
Here’s the problem in a nutshell: this is a bincode deserialization error.
The issue is that here, we’re trying to read a json file.

Reply:
This is now fixed by #511. Please re-instate.